src/librustc_error_codes/error_codes/E0752.md

```diff
@@ -1,11 +1,19 @@
-`fn main()` or the specified start function is not allowed to be
-async. You might be seeing this error because your async runtime
-library is not set up correctly.
+The entry point of the program was marked as `async`.
 
 Erroneous code example:
 
 ```compile_fail,E0752
-async fn main() -> Result<i32, ()> {
+async fn main() -> Result<i32, ()> { // error!
+    Ok(1)
+}
+```
+
+`fn main()` or the specified start function is not allowed to be `async`. You
+might be seeing this error because your async runtime library is not set up
+correctly. To fix it, don't declare the entry point as `async`:
```

Comment:
```suggestion
`fn main()` or the specified start function is not allowed to be `async`. Not
having a correct async runtime library setup may cause this error. To fix it,
declare the entry point without `async`:
```

Change to active wording, remove the use of "you".

Do we need "the specified start function"? Is `start` redirected to this error too?